tox.ini

```diff
@@ -79,10 +79,10 @@ envlist =
     {py3.7}-gcp
 
     # HTTPX
-    {py3.6,py3.7,py3.8,py3.9,py3.10,py3.11}-httpx-v{0.16,0.17}
```

Comment:
why can't we keep testing the older versions?

Reply:
no reason. but you are right, I added them again. All the versions 0.16+